frontend/src/app/shared/components/global-footer/global-footer.component.scss

```diff
@@ -88,7 +88,14 @@ footer .row.link-tree {
 
 footer .row.social-links {
   text-align: center;
-  margin: 24px 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: fit-content;
+  margin: 0 auto; 
+
+  @media (max-width: 450px){
+    width: 250px;
```

Comment:
For narrow screens set flex-box to a fixed width forcing a row of 4 on top and 3 on bottom